src/guide/profiles.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Profiles
+type: guide
+order: 6
+---
+
+Infection supports the use of mutator profiles for the command line and configuration file.
+
+The following configuration file will use the `default` profile, but turn off the `function_signature` profile. 
+On top of that, it does not apply the `TrueValue` mutator on any classes that match the provided ignore patterns.
+These ignores can also be added to profiles, to ensure infection is as flexible as you need it. 
+
+All profiles are prepended by an `@` and in snake case, while all mutators are in PascalCase. 
+
+``` json
+{
+    "source": {
+        "directories": [
+            "src"
+        ],
+    },
+    "timeout": 10,
+    "logs": {
+        "text": "infection-log.txt",
+    }
+    "mutators": {
+        "@default": true,
+        "@function_signature": false,
+        "TrueValue": {
+            "ignore": [
+                "NameSpace\\*\\SourceClass::method",
+                "Full\\NameSpaced\\Class"
+            ]
+        }
+    }
+}
+```
+
+### The Profiles
+
+Currently infection supports the following profiles:
```

Comment:
Probably, there should be a comma after `Currently`. At least Grammarly plugin reports an issue